nsi script/script.nsi

```diff
@@ -0,0 +1,7 @@
+Outfile "logicrs-windows-x86_64.exe"
+Section
+    StrCpy $INSTDIR $EXEDIR
+    StrCpy $INSTDIR "$INSTDIR\logicrs"
+    SetOutPath $INSTDIR
+    File /r "F:\exe\logicrs-windows-x86_64\*.*"
```

Comment:
Can this be a relative path?

Reply:
just so i understand it, the line File /r will fetch all the files from that directory and you want it to be relative? Is that it?

Reply:
If you ask me, File /r "path\*.*" is fine with being absolute, rather it's good it's that way. 
Using $EXEDIR is more important since it fetches the current directory path.
If you want to try a relative path, can you specify any example directory so that I can work it out?

Reply:
So, I have made a script for relative path and here's the sample directory I used:

![image](https://github.com/Spydr06/logicrs/assets/80760499/2f83d6c9-7707-4548-9661-617ca572704a)
![image](https://github.com/Spydr06/logicrs/assets/80760499/f2746ed8-246b-4846-868f-daae24d95d87)

When you load the nsi script in NSIS, it will generate the .exe you see in the 2nd image.

I have tested it out in a different folder, and when we double click it will create a new folder logicrs and install the files in that: 
![image](https://github.com/Spydr06/logicrs/assets/80760499/151a47db-d1f2-4a22-8233-0f08ebbaef3a)


Can I push this code or do you want more changes? @Spydr06

Reply:
Yes that way it's better, looks good :)

Reply:
Ok, I will commit the new script

Reply:
Please check the PR, thanks.

Reply:
Somehow the deleted commit didn't took place in my previous commit, I have just fixed it.